packages/nvidia_nat_eval/src/nat/plugins/eval/runtime/eval_harness.py

```
@@ -0,0 +1,88 @@
+        """
+        atif_evaluate_fn = getattr(evaluator, "evaluate_atif_fn", None)
+        if not callable(atif_evaluate_fn):
+            self._logger.warning("Skipping evaluator %s: missing callable evaluate_atif_fn", evaluator_name)
+            return None
+
+        try:
+            eval_output = await atif_evaluate_fn(atif_samples)
+            return evaluator_name, eval_output
+        except Exception as e:
+            # Best-effort policy: log per-evaluator failure and continue.
+            self._logger.exception("An error occurred while running evaluator %s: %s", evaluator_name, e)
```

Ruff (0.15.4)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)